spec/main_game_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# require_relative '../lib/game_play.rb'
+
+# RSpec.describe '::Game' do
+#   let(:game) { Game.new }
+
+#   describe '#initialize' do
+#     it 'initializes a new game object' do
+#       expect(game.nil?).to eql(false)
+#     end
+#   end
+# end
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.